src/http/auth.js

```diff
@@ -0,0 +1,8 @@
+import { httpClient } from "@/http/httpClient";
+import { REGISTER_ENDPOINT, AUTHENTICATE_ENDPOINT } from "@/http/ENDPOINTS";
+
+const signin = ({ username, password }) =>
+  httpClient.post(AUTHENTICATE_ENDPOINT, { username, password });
+
+const register = (userInfor) => httpClient.post(REGISTER_ENDPOINT, userInfor);
+export { signin, register };
```

Comment:
```suggestion

export {
  signin,
  register,
};

```
that helps us avoid conflict.